department-of-veterans-affairs/caseflow
# Judge algorithm fixes for Caseflow parent records

Connects #8008 

### Description
This PR updates the judge algorithm to work correctly with parent records in Caseflow.

### Testing Plan
1. Create CO hearing days
1. Run the judge algorithm
1. Confirm results look good!
